### PR TITLE
Feature/add hosted singlenode

### PR DESCRIPTION
**Pull request by clstrfsck**

### CODE REVIEW

File: src/modules/universe/RadixNodeDiscoveryHardcodedSecure.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    public async loadNodes() {
+        return Promise.all(this.bootstrapNodes.map(async (address) => {
+            try {
+                const response = await axios.get(`https://${address}/api/system`)
+                const nodeInfo: RadixNodeSystem = RadixSerializer.fromJSON(response.data)
+                const node = new RadixNode(nodeInfo, `wss://${address}/rpc`, `https://${address}/rpc`)
+                return node
+            } catch (error) {
+                throw error
```
Comment (laszlourszuly) [inline src/modules/universe/RadixNodeDiscoveryHardcodedSecure.ts:29]: Why catching if we just throw it again? Why the `try` at all?

File: src/modules/universe/RadixNodeDiscoveryHardcodedSecure.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+     * @param bootstrapNode Full address to the rpc endpoint of a Radix node in the universe
+     */
+    constructor(
+        readonly bootstrapNodes: string[],
+        ) {}
+
+    public async loadNodes() {
+        return Promise.all(this.bootstrapNodes.map(async (address) => {
+            try {
+                const response = await axios.get(`https://${address}/api/system`)
```
Comment (laszlourszuly) [inline src/modules/universe/RadixNodeDiscoveryHardcodedSecure.ts:24]: I may be mixing things up now, but I'll ask anyway (just for my own peace of mind): Are we sure about this endpoint? The `BootstrapByTrustedNode` implementation in the Java library uses the `/api/universe` endpoint for example. This is also where I suspect I may compare apples to pears.

Reference: https://github.com/radixdlt/radixdlt-java/blob/release/1.0.0-beta/radixdlt-java/src/main/java/com/radixdlt/client/core/BootstrapByTrustedNode.java#L31

**Review by laszlourszuly — COMMENTED** [review]
Looks good. Only one minor comment regarding a try/catch block. And also a stupid question regarding api endpoints.

File: src/modules/universe/RadixNodeDiscoveryHardcodedSecure.ts (hunk 2 above)
Comment (clstrfsck) [reply src/modules/universe/RadixNodeDiscoveryHardcodedSecure.ts:24]: Yes, this is a different mechanism — for node discovery.

The universe is hard coded in the JS library right now, so no universe discovery yet.

File: src/modules/universe/RadixNodeDiscoveryHardcodedSecure.ts (hunk 1 above)
Comment (clstrfsck) [reply src/modules/universe/RadixNodeDiscoveryHardcodedSecure.ts:29]: I don't know :(

Copy/plastered from RadixNodeDiscoveryHardcoded.ts and adjusted.  I'll add a task in the backlog to get this consolidated into a single piece of code once someone with more typescript-fu than me has time.

**Pull request merged**